relate/src/main/scala/com/lucidchart/relate/RowParser.scala

```diff
@@ -25,24 +25,4 @@ object RowParser extends CollectionsParser {
 
   private[relate] val insertInt = (row: SqlRow) => row.strictInt(1)
   private[relate] val insertLong = (row: SqlRow) => row.strictLong(1)
-
-  implicit def multiMap[Key: RowParser, Value: RowParser] = RowParser[Map[Key, Set[Value]]] { result =>
```

Comment:
why are we getting rid of this?

Reply:
I get that it shouldn't be part of `RowParser`, but should we add something equivalent to `CollectionsSqlResult`?